### PR TITLE
Changing county summaries to iterate over model output files to pickup fips

### PR DESCRIPTION
**Pull request by ghop02**
Fixes https://github.com/covid-projections/covid-projections/issues/534

### CODE REVIEW

File: libs/pipelines/api_pipeline.py [hunk 1]
```diff
@@ -179,6 +183,37 @@ def build_counties_timeseries(
...
+    """
+    found_fips_by_state = defaultdict(set)
+    found_fips = set()
+    for path in pathlib.Path(input_dir).iterdir():
+        if not str(path).endswith('.json'):
+            continue
+
+        state, fips, intervention, _ = path.name.split('.')
+        found_fips_by_state[state].add(fips)
+        found_fips.add(fips)
```
Comment (goldblatt) [inline libs/pipelines/api_pipeline.py:202]: this feels a lil ruff don't we have access to this as a var somewhere?

File: libs/pipelines/api_pipeline.py [hunk 2]
```diff
@@ -179,6 +183,37 @@ def build_counties_timeseries(
...
+
+    Returns: List of API Output objects.
+    """
+    found_fips_by_state = defaultdict(set)
+    found_fips = set()
+    for path in pathlib.Path(input_dir).iterdir():
+        if not str(path).endswith('.json'):
+            continue
+
+        state, fips, intervention, _ = path.name.split('.')
```
Comment (goldblatt) [inline libs/pipelines/api_pipeline.py:200]: also does python have the like match stuff (I'm thinking in javascript right now), but like a regex match?

File: libs/pipelines/api_pipeline.py (hunk 1 above)
Comment (ghop02) [reply libs/pipelines/api_pipeline.py:202]: I could load all of the data files, but don't need anything inside them so this was the easiest way to get this dat

File: libs/pipelines/api_pipeline.py (hunk 2 above)
Comment (ghop02) [reply libs/pipelines/api_pipeline.py:200]: yeah it does - could do it later but this works well enough for right now

**Pull request merged**